src/modules/services/submodule/nft/views/ViewServicesNFT.vue

```diff
@@ -4,7 +4,7 @@
       <div class="flex">
         <div class='py-3 px-6 lg:flex items-center'>
           <div class="text-xl mr-2 mb-2">NFT</div>
-          <MultiDropdownPortfolioAccountComponent :account="accounts" @checked='onCheck' />
+          <MultiDropdownPortfolioAccountComponent :account?="accounts" @checked='onCheck' />
```

Comment:
what is this question mark for?

Reply:
Question mark is used to define an argument as optional. It is the same as specifying the type as undefined. Else it will appear error which the compiler considers account as a null value or undefined value. So, this can be solve by adding an question mark after the account. (Website answer)